Fix

staeff opened the pull request:
Fixes:  
* ConfigurationError on startup of the instance. This was due to the use of the attribute 'post_handler' in configure.zcml. Version 1.8.0 of GenericSetup is currently pinned. The attributes are introduced at version 1.8.2. (Unfortunately upgrading to 1.8.2. causes a different problem. So this is currently something that works)
Improves:  
* Removes generated files from version control.

vikas-parashar commented:
@staeff There is fix for GenericSetup version problem, In line 2 in`buildout.cfg` changing http://dist.plone.org/release/5.0/versions.cfg to extends = http://dist.plone.org/release/5.0.5/versions.cfg fixes the issue of version constraint. Maybe you can try that and see if the error is fixed. Then we won't need to edit `configure.zcml`

staeff commented:
@vikas-parashar I updated pull request. As I was saying, just using 5.0.5s version.cfg was not sufficient to fix everything. Currently its still needed to avoid 'post_handler' (I try to figure out the problem).
You can check with `bin/test -s plonetheme.booster` that the code of the PR is running fine, but with post_handler is failing.
Additionally I fixed the postinstall steps to actually run.

vikas-parashar commented:
@staeff thanks
